mlir-tensorrt/executor/include/mlir-executor/Runtime/API/API.h

```diff
@@ -120,7 +120,7 @@ std::string_view stringifyPointerType(PointerType ptrType);
 //===----------------------------------------------------------------------===//
 // TypeView
 // This section includes classes that form the TypeUnion:
-// MemrefTypeView, ScalarTypeView, ExternalOpaqueTypeView
+// MemRefTypeView, ScalarTypeView, ExternalOpaqueTypeView
```

Comment:
Please make NFC rename changes a separate PR or at least a separate commit in the PR. Isolating NFC changes is always better since it lowers the review burden.

Reply:
Sure. I will do so.